docs/multi-instance.md

```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
 
 If you have single-instances hosts and want to upgrade from previous versions of the role:
 
+### Procedure with datas move
```

Comment:
`datas` -> `data`, this is in a few places.

Reply:
@Conky5 fixed